tests/upstream_switch/case.sh

```diff
@@ -217,6 +227,48 @@ function test() {
 	gen_incr_data
 	verify_result
 	clean_task
+	echo "CASE=test_relay success"
+}
+
+function test_binlog_filename_change_when_enable_gtid() {
+	cleanup_process
+	check_master
+	clean_data
+	setup_replica
+	gen_full_data
+	run_dm_components_and_create_sources
+	start_task
+
+	# flush mysql8 master binlog files -> mysql-bin.000002
+	exec_sql $host1 "flush logs;"
+	# flush mysql8 master binlog files -> mysql-bin.000004
```

Comment:
one is master, one is slave?

Reply:
$host1 is svip of two mysql and it's always point to master